.github/workflows/test-status-check.yml

```
@@ -15,20 +15,26 @@ on:
 jobs:
   main_tests:
     name: Run ${{ vars.PROJECT_NAME }} Tests
-    uses: KinsonDigital/Infrastructure/.github/workflows/run-csharp-tests.yml@v13.0.1
+    uses: KinsonDigital/Infrastructure/.github/workflows/run-csharp-tests.yml@v13.1.0
     with:
       project-name: "${{ vars.PROJECT_NAME }}Tests"
       build-config: Debug
       net-sdk-version: "${{ vars.NET_SDK_VERSION }}"
+      checkout-ref: ${{ github.event.pull_request.head.ref }}
+      checkout-repository: ${{ github.event.pull_request.head.repo.full_name }}
+
 
 
   integration_tests:
     name: Run ${{ vars.PROJECT_NAME }} Integration Tests
-    uses: KinsonDigital/Infrastructure/.github/workflows/run-csharp-tests.yml@v13.0.1
+    uses: KinsonDigital/Infrastructure/.github/workflows/run-csharp-tests.yml@v13.1.0
     with:
       project-name: "${{ vars.PROJECT_NAME }}IntegrationTests"
       build-config: Debug
       net-sdk-version: "${{ vars.NET_SDK_VERSION }}"
+      checkout-ref: ${{ github.event.pull_request.head.ref }}
+      checkout-repository: ${{ github.event.pull_request.head.repo.full_name }}
+
```

Please remove the extra line so that end result is only 2 empty lines between jobs.

```suggestion
      checkout-ref: ${{ github.event.pull_request.head.ref }}
      checkout-repository: ${{ github.event.pull_request.head.repo.full_name }}
```